src/components/alert/alert-component.ts

```diff
@@ -128,6 +140,29 @@ export class AlertCmp {
     }
   }
 
+  ionViewPreLoad() {
+    if (this.d.subview && this.d.subview.component) {
```

Comment:
Can we change this to his return-early way? https://github.com/freeCodeCamp/freeCodeCamp/wiki/Return-Early-Pattern
```ts
if (!this.d.subview || !this.d.subview.component) {
  return;
}
```